test/e2e/framework/test_context.go

```diff
@@ -57,14 +57,16 @@ type TestContextType struct {
 	Prefix         string
 	MinStartupPods int
 	// Timeout for waiting for system pods to be running
-	SystemPodsStartupTimeout time.Duration
-	UpgradeTarget            string
-	EtcdUpgradeStorage       string
-	EtcdUpgradeVersion       string
-	UpgradeImage             string
-	GCEUpgradeScript         string
-	ContainerRuntime         string
-	ContainerRuntimeEndpoint string
+	SystemPodsStartupTimeout    time.Duration
+	UpgradeTarget               string
+	EtcdUpgradeStorage          string
+	EtcdUpgradeVersion          string
+	UpgradeImage                string
+	GCEUpgradeScript            string
+	ContainerRuntime            string
+	ContainerRuntimeEndpoint    string
+	ContainerRuntimeProcessName string
+	ContainerRuntimePidFile     string
```

Comment:
Maybe `ContainerRuntimeProcessName` and `ContainerRuntimePidFile` need to be a list or map, since some runtimes contain several binaries that belong to different cgroups (e.g.  containerd and cri-cointainerd)

Reply:
@miaoyq You can do that in your benchmark PR if you find that is required. :)